content/operation/known_issues.md

```diff
@@ -7,64 +7,86 @@ chapter: false
+
+### Support for `application/soap+xml`
+
+As of CRS version 3.0.1, support has been added for the `application/soap+xml` MIME type by default, as specified in RFC 3902. **OF IMPORTANCE:** application/soap+xml is indicative that XML will be provided. In accordance with this, ModSecurity's XML request body processor should also be configured to support this MIME type. Within the ModSecurity project, [commit 5e4e2af](https://github.com/owasp-modsecurity/ModSecurity/commit/5e4e2af7a6f07854fee6ed36ef4a381d4e03960e) has been merged to support this endeavor. However, if running a modified or preexisting version of the modsecurity.conf file provided by this repository, it is a good idea to upgrade rule '200000' accordingly. The rule now appears as follows:
+
+```
+SecRule REQUEST_HEADERS:Content-Type "(?:application(?:/soap\+|/)|text/)xml" \
+  "id:'200000',phase:1,t:none,t:lowercase,pass,nolog,ctl:requestBodyProcessor=XML"
+```
+
+### `SecDisableBackendCompression` not supported in libmodsecurity3
+
+All versions of libmodsecurity3 [do not support](https://github.com/owasp-modsecurity/ModSecurity/wiki/Reference-Manual-(v3.x)#secdisablebackendcompression) the `SecDisableBackendCompression` directive at all ###
 
-  ```
-  SecRule REQUEST_HEADERS:Content-Type "(?:application(?:/soap\+|/)|text/)xml" \
-    "id:'200000',phase:1,t:none,t:lowercase,pass,nolog,ctl:requestBodyProcessor=XML"
-  ```
-- **All versions of libmodsecurity3** [do not support](https://github.com/owasp-modsecurity/ModSecurity/wiki/Reference-Manual-(v3.x)#secdisablebackendcompression) the `SecDisableBackendCompression` directive at all. 
 If Nginx is acting as a proxy and the backend supports any type of compression, if the client sends an `Accept-Encoding: gzip,deflate,...` or `TE` header, the backend will return the response in a compressed format. Because of this, the engine cannot verify the response. As a workaround, you need to override the `Accept-Encoding` and `TE` headers in the proxy:
 
-   ```
-    server {
-        server_name     foo.com;
+```nginx
+server {
+    server_name     foo.com;
+    ...
+    location / {
+        proxy_pass         http://backend;
         ...
-        location / {
-            proxy_pass         http://backend;
-            ...
-            proxy_set_header   Accept-Encoding "";
-            proxy_set_header   TE "";
-        }
+        proxy_set_header   Accept-Encoding "";
+        proxy_set_header   TE "";
     }
-   ```
+}
+```
+
+### Anomaly Mode Doesn't Work
+
+Sometimes on IIS or Nginx users run into an instance where anomaly mode doesn't work as expected. In fact upon careful inspection of logs one would notice that rules don't fire in the order we would expect. In general this is a result of using the `'*'` operator within these environments as it does not act the same way as in Apache. In general within both Apache and IIS one should expliticly include the various files present within the OWASP CRS instead of using the `'*'`.
```

Comment:
`In general within both Apache and IIS` should probably be `Nginx and IIS`.

Also: Does this problem persist with newer versions? Because if it does, our INSTALL that proposes `*` now is likely wrong.

Reply:
Sadly, I do not have a clue. This was copied from the Website 🤷 . Maybe we need to clean this out?

Reply:
@airween what is the real status of `Include rules/*.conf` on Nginx?

Reply:
Sorry, what do you mean about "real status"?

The mentioned configuration (`Include rules/*.conf`) on Nginx works as well, I use that form all of my servers.

Reply:
I thought so. That means this passage here can be omitted and we should update the website too.